tests/src/main/java/alluxio/master/backcompat/ops/PersistFile.java

```diff
@@ -37,10 +38,10 @@ public void apply(Clients clients) throws Exception {
     FileSystem fs = clients.getFileSystemClient();
     Utils.createFile(fs, PATH);
     clients.getFileSystemMasterClient().scheduleAsyncPersist(PATH,
-        ScheduleAsyncPersistencePOptions.getDefaultInstance());
```

Comment:
FileSystem should do merging.

Reply:
Not here, we don't use a `FileSystem` object. Options must be specified to this client.

Reply:
Right. You're calling the client. My bad.
I was consumed by Copy/Paste rage...

Reply:
It happens to the best of us